cellarium/cas/models.py

```diff
@@ -0,0 +1,188 @@
+import typing as t
+
+from pydantic import BaseModel, Field
+
+
+class CellTypeSummaryStatisticsResults(BaseModel):
+    """
+    Represents the data object returned by the CAS API for nearest neighbor annotations.
+    """
+
+    class DatasetStatistics(BaseModel):
```

Comment:
Possibly a dumb question: Are these classes nested because they are only used for fields of `CellTypeSummaryStatisticsResults`?

Reply:
Yeah that's exactly right